README.md

```diff
@@ -17,6 +17,21 @@ paths=""
 services=""
 </pre>
 
+### proxy_idp_auth_test.sh
+* Attributes to be filled:
+<pre>
+# The url of tested SP
+# For example :https://aai-playground.ics.muni.cz/simplesaml/nagios_check.php?proxy_idp=cesnet
+testSite=""
+
+loginSite="https://idp2.ics.muni.cz/idp/Authn/UserPassword"
+
+# Fill in UCO as number
+uco=
```

Comment:
Please rename it to login. The probe should be generic.

Reply:
Done